vendor.mod

```diff
@@ -6,14 +6,11 @@ module github.com/docker/cli
 
 go 1.18
 
-// use a replace rule to prevent go mod downgrading (because master gets v24.0.0-rc.2.xxx pseudo-versions)
-replace github.com/docker/docker => github.com/docker/docker v24.0.0-rc.2.0.20230506131059-88f4bf4ae4ba+incompatible
-
```

Comment:
Looks like it's (so far) still happy without the replace rule, so let's remove it for now